### ISSUE TITLE
Unexpected token and property errors

### ISSUE THREAD

**Issue by atmediauk** [optimistdigital/nova-multiselect-field#6]
Hello, hoping you can help. When creating a field with the following code:

```
            Multiselect::make('Sectors', 'sectors')
                ->rules('required')
                ->options([
                    'Option1' => 'Option1',
                    'Option2' => 'Option2',
                    'Option3' => 'Option3',
                ])
                ->placeholder('Choose')
                ->max(5),
```

I receive a  console error on page load:
```
SyntaxError: Unexpected token e in JSON at position 0
    at JSON.parse 
```

If I proceed to click save and continue anyway I then get a new error:

```
TypeError: Cannot read property 'status' of undefined
```

I cleared my browser cache, Laravel cache and also disabled all the other fields but the error remains. As soon as I remove the multiselect the page starts to load and save correctly again.

Any ideas on what could be causing this?

**Comment by goinnovative** [comment]
Same problem here...

**Comment by Tarpsvo** [comment]
Hey! Thanks for reporting this @atmediauk and thanks @goinnovative for an accurate and helpful guess at what the problem is. I added a `try-catch` around the `JSON.parse()` function and if it crashes, I'll just reset the field value to an empty array (`[]`) as the data in the database may be invalid.

The fix was published in version 1.1.5.
